web/src/features/hfiCalculator/components/WeeklyViewTable.tsx

```diff
@@ -138,9 +150,30 @@ export const WeeklyViewTable = (props: Props): JSX.Element => {
                         key={`zone-${areaName}`}
                         data-testid={`zone-${areaName}`}
                       >
-                        <TableCell className={classes.planningArea} colSpan={5}>
-                          {area.name}
-                        </TableCell>
+                        <StickyCell
+                          left={0}
+                          zIndexOffset={10}
+                          backgroundColor={'#e9ecf5'}
```

Comment:
For other parameters of sticky cells like `left`, etc, they are usually non-uniform. This `backgroundColor` is repeated a bunch though and probably can be named as a variable.